command.sh

```
@@ -0,0 +1,8 @@
+ sudo cat /sys/fs/cgroup/nexus/node-alpine/cgroup.procs
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)